cypress/support/kaoto-ui-commands/steps.js

```diff
@@ -64,7 +64,7 @@ Cypress.Commands.add('deleteStep', (step, stepIndex) => {
         const previousStepsCount = elem.length;
 
         /** Delete the given step */
-        cy.get(`[data-testid="viz-step-${step}"]`).eq(stepIndex).trigger('mouseover').children('[data-testid="configurationTab__deleteBtn"]').click({ force: true });
+        cy.get(`[data-testid="viz-step-${step}"]`,{withinSubject:null}).eq(stepIndex).trigger('mouseover').children('[data-testid="configurationTab__deleteBtn"]').click({ force: true });
```

Comment:
Is this `{withinSubject: null}` ok or would be better to have a dedicated handler from the UI side?

Reply:
found this is probably redundant - removed, thanks for noticing this